model/utils.go

```diff
@@ -70,6 +71,13 @@ type AppError struct {
 }
 
 func (er *AppError) Error() string {
+	// below is a short-circuit to avoid crashing during error reporting
+	// if by some chance the error is nil, we don't want crash, but we do report the problem to Sentry
+	if er == nil {
+		msg := "Error() is called on a nil AppError instance"
+		sentry.CaptureMessage(msg)
```

Comment:
Will we still get a stack trace indicating where in the code this was invoked by the nil error? Or does this just capture the text of the message above?

Reply:
this captures the stacktrace as well